Usage instructions and example give "can't find crate for `msgpack`"

I'm new to rust. I followed the instructions in the README to build the example given there. I'm running a rust nightly.

The build gives this warning, which seems like it might by relevant: (full output below)

```
dependency (rmp) specified without providing a local path, Git repository, or version to use. This will be considered an error in future versions
```

# My `rustc --version`:

```
$ rustc --version
rustc 1.14.0-nightly (5665bdf3e 2016-11-02)
```

# My `Cargo.toml`:

```toml
[package]
name = "msgpack-hello-world"
version = "0.0.0"

[dependencies.rmp]
rmp = "^0.7"
```

# Running `cargo build`

```
$ cargo build
warning: dependency (rmp) specified without providing a local path, Git repository, or version to use. This will be considered an error in future versions
warning: unused manifest key: dependencies.rmp.rmp
   Compiling msgpack-hello-world v0.0.0 (file:///home/pwaller/Projects/msgpack-hello-world)
error[E0463]: can't find crate for `msgpack`
 --> src/main.rs:1:1
  |
1 | extern crate rmp_serialize as msgpack;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error

error: Could not compile `msgpack-hello-world`.

To learn more, run the command again with --verbose.
```

--- a/README.md
+++ b/README.md
@@ -19,9 +19,12 @@ deserializing Rust structs.
 To use `rmp`, first add this to your `Cargo.toml`:
 
 ```toml
-[dependencies.rmp]
-rmp = "^0.7"
+[dependencies]
+rmp = "0.7"
 ```
+
+For [rustc-serialize] serialization support, add `rmp-serialize = "0.7"` to your
+dependencies. For [Serde] serialization, add `rmp-serde = "0.10"`.
 
 Then, add this to your crate root:
 
@@ -54,7 +57,8 @@ extern crate rmp as msgpack; // Or just `rmp`.
 
 ## Examples
 
-Let's try to encode a tuple of int and string.
+Let's try to encode a tuple of int and string. These examples assume you are
+using the `rustc-serialize` serialization crate.
 
 ```rust
 extern crate rmp_serialize as msgpack;
@@ -138,6 +142,9 @@ will be the following rules:
    version bumping.
  - API extending features results in patch version bumping.
  - Non-breaking bug fixes and performance improving results in patch version bumping.
+
+[rustc-serialize]: https://github.com/rust-lang-nursery/rustc-serialize
+[serde]: https://github.com/serde-rs/serde
 
 [coveralls-img]: https://coveralls.io/repos/3Hren/msgpack-rust/badge.svg?branch=master&service=github
 [coveralls-url]: https://coveralls.io/github/3Hren/msgpack-rust?branch=master

